src/main/java/org/generationcp/middleware/pojos/dms/StockModel.java

```diff
@@ -93,6 +93,10 @@
 	@Column(name = "is_obsolete")
 	private Boolean isObsolete;
 
+	@Basic(optional = false)
+	@Column(name = "project_id")
+	private Integer projectId;
```

Comment:
I think the attribute should be DmsProject type instead of integer and keep the relationship with project table.